### ISSUE TITLE
Banking app crashes with LawnStep 1.1.1

### ISSUE THREAD

**Issue by runekock** [LawnchairLauncher/LawnstepModule#2]
My banking app needs Magisk hide to run. It works fine with LawnStep 1.0.4, but if I install 1.1.1, the banking app crashes with this message in logcat:

[8601,0,dk.bec.android.mb1.b00888.prod,948485700,java.io.FileNotFoundException,File doesn't exist: /system/vendor/overlay/Lawnstep.apk,ZipFile.java,215]

**Comment by skittles9823** [comment]
I'm so confused. I'll keep note of this though. Also, be aware, the lawnstep module will be deprecated soon for a better solution, so perhaps you'll not have the same issue there.

**Comment by runekock** [comment]
Yeah, it's not clear what that banking app is doing. My guess is that they try to scan all installed apps to look for malware. Could it be that Magisk hide partially hides LawnStep, so that the banking app thinks it should be there, but can't see it? Anyway, I'll try your new solution when it's ready.

**Comment by skittles9823** [comment]
Magisk essentially unmounts all modules when an app that has magisk hide enabled is opened so I really doubt it's aware of lawnstep unless its specifically searching for it. But the new module has all the files renamed for ease and compatibility so it'll be interesting to see if it'll work with the banking app.
